packages/webpack/src/utils/postcss.ts

```diff
@@ -55,20 +57,35 @@ export const getPostcssConfig = (nuxt: Nuxt) => {
     },
     sourceMap: nuxt.options.webpack.cssSourceMap,
     // Array, String or Function
-    order: 'autoprefixerAndCssnanoLast',
+    order: 'autoprefixerAndCssnanoLast' as const,
   })
 
   // Keep the order of default plugins
   if (!Array.isArray(postcssOptions.plugins) && isPureObject(postcssOptions.plugins)) {
     // Map postcss plugins into instances on object mode once
     const cwd = fileURLToPath(new URL('.', import.meta.url))
-    postcssOptions.plugins = sortPlugins(postcssOptions).map((pluginName: string) => {
-      // TODO: remove use of requireModule in favour of ESM import
-      const pluginFn = requireModule(pluginName, { paths: [cwd] })
+    const plugins: Plugin[] = []
+    for (const pluginName in sortPlugins(postcssOptions)) {
       const pluginOptions = postcssOptions.plugins[pluginName]
-      if (!pluginOptions || typeof pluginFn !== 'function') { return null }
-      return pluginFn(pluginOptions)
-    }).filter(Boolean)
+      if (!pluginOptions) { continue }
+
+      const path = await tryResolveModule(pluginName, nuxt.options.modulesDir)
+
+      let pluginFn: (opts: Record<string, any>) => Plugin
+      if (path) {
+        pluginFn = await import(pathToFileURL(path).href).then(interopDefault)
+      } else {
+        console.warn(`[nuxt] could not import postcss plugin \`${pluginName}\` with ESM. Please report this as a bug.`)
+        // fall back to cjs
+        pluginFn = requireModule(pluginName, { paths: [cwd] })
```

Comment:
Migrating to jiti v2, this can all be done automatically and it always prefers native ESM with fallback.

Reply:
Good point.

Will need to test to see if there are any Potential Side Effects and Breaking Changes.

than you